docs/latest/concepts/islands.md

```diff
@@ -4,36 +4,36 @@ description: |
 ---
 
 Islands enable client side interactivity in Fresh. Islands are isolated Preact
-components that are rendered on the client. This is different from all other
-components in Fresh, as they are usually just rendered on the server.
+components that are rendered on the server and then hydrated on the client. This
+is different from all other components in Fresh, as they are usually rendered on
+the server only.
 
 Islands are defined by creating a file in the `islands/` folder in a Fresh
 project. The name of this file must be a PascalCase or kebab-case name of the
 island.
 
 ```tsx islands/my-island.tsx
-import { useSignal } from "@preact/signals";
+import {useSignal} from '@preact/signals';
 
 export default function MyIsland() {
-  const count = useSignal(0);
-
-  return (
-    <div>
-      Counter is at {count}.{" "}
-      <button onClick={() => (count.value += 1)}>+</button>
-    </div>
-  );
+	const count = useSignal(0);
+
+	return (
+		<div>
+			Counter is at {count}. <button onClick={() => (count.value += 1)}>+</button>
+		</div>
+	);
```

Comment:
It looks like `deno fmt` isn't able to format fenced code blocks with additional arguments.

Reply:
Filed an issue https://github.com/denoland/deno/issues/20309

Reply:
@iuioiua Should i do something about that now or not? 
And if yes, what should I do about the code? 

I tried configuring deno as the default formatter for markdown files (in case vscode accidentally chose my personal default: prettier). 
But this did not change anything (as expected from your FYI to deno issue 16160). 
I'd suggest to keep it like this until deno fmt is fixed, otherwise this would be inconsistent with deno fmt.

Reply:
You can revert any code formatting change unrelated to the purpose of this PR by just copy/pasting from the main branch. You may have to disable your formatter temporarily.